core/trino-main/src/main/java/io/trino/testing/TestingConnectorSession.java

```diff
@@ -151,10 +151,10 @@ public static Builder builder()
     public static class Builder
```

Comment:
typo in commit message `itteration`

I think we should focus on all queries, not only SELECT.

Please also state some motivation in the commit message about why it is needed.